### ISSUE TITLE
Using pen's width > 1 with a single point plot cause error

### ISSUE THREAD

**Issue by msmttchr** [pyqtgraph/pyqtgraph#2593]
### Short description
In a plot with single point using a pen with width>1 causes exception.
Even though this is something not usual, it will apply in cases when the plot is built live from data collected real-time.

The exception shown is:
```

C:\GIT\pymeasure\examples\Basic>py -3 bug.py
C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\debug.py:128: RuntimeWarning: Ignored exception:
Traceback (most recent call last):
  File "C:\GIT\pymeasure\examples\Basic\bug.py", line 94, in <module>
    pg.exec()
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\Qt\__init__.py", line 388, in exec_
    return app.exec() if hasattr(app, 'exec') else app.exec_()
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\widgets\GraphicsView.py", line 137, in paintEvent
    return super().paintEvent(ev)
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\debug.py", line 128, in w
    printExc('Ignored exception:')
  --- exception caught here ---
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\debug.py", line 126, in w
    func(*args, **kwds)
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\graphicsItems\PlotCurveItem.py", line 905, in paint
    p.drawLines(*self._getLineSegments())
TypeError: arguments did not match any overloaded call:
  drawLines(self, PyQt6.sip.array[QLineF]): not enough arguments
  drawLines(self, QLineF, *): not enough arguments
  drawLines(self, PyQt6.sip.array[QPointF]): not enough arguments
  drawLines(self, QPointF, *): not enough arguments
  drawLines(self, PyQt6.sip.array[QLine]): not enough arguments
  drawLines(self, QLine, *): not enough arguments
  drawLines(self, PyQt6.sip.array[QPoint]): not enough arguments
  drawLines(self, QPoint, *): not enough arguments
  printExc('Ignored exception:')
C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\debug.py:128: RuntimeWarning: Ignored exception:
Traceback (most recent call last):
  File "C:\GIT\pymeasure\examples\Basic\bug.py", line 94, in <module>
    pg.exec()
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\Qt\__init__.py", line 388, in exec_
    return app.exec() if hasattr(app, 'exec') else app.exec_()
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\widgets\GraphicsView.py", line 137, in paintEvent
    return super().paintEvent(ev)
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\debug.py", line 128, in w
    printExc('Ignored exception:')
  --- exception caught here ---
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\debug.py", line 126, in w
    func(*args, **kwds)
  File "C:\Users\michele sardo\AppData\Local\Programs\Python\Python310\lib\site-packages\pyqtgraph\graphicsItems\PlotCurveItem.py", line 905, in paint
    p.drawLines(*self._getLineSegments())
TypeError: arguments did not match any overloaded call:
  drawLines(self, PyQt6.sip.array[QLineF]): not enough arguments
  drawLines(self, QLineF, *): not enough arguments
  drawLines(self, PyQt6.sip.array[QPointF]): not enough arguments
  drawLines(self, QPointF, *): not enough arguments
  drawLines(self, PyQt6.sip.array[QLine]): not enough arguments
  drawLines(self, QLine, *): not enough arguments
  drawLines(self, PyQt6.sip.array[QPoint]): not enough arguments
  drawLines(self, QPoint, *): not enough arguments
  printExc('Ignored exception:')
```
### Code to reproduce
See attached file
### Tested environment(s)

 * PyQtGraph version: 
 * Qt Python binding: 0.13.1
 * Python version: 3.10
 * NumPy version: 1.23.4
 * Operating system: Windows 10
 * Installation method: pip

### Additional context
[bug.zip](https://github.com/pyqtgraph/pyqtgraph/files/10510794/bug.zip)

**Comment by pijyoi** [comment]
Fixed in #2481

**Comment by msmttchr** [comment]
Thanks for the quick answer
